src/_pytest/config/__init__.py

```diff
@@ -14,7 +14,6 @@
 
 import py
 import six
-from pkg_resources import parse_version
```

Comment:
I'm -0

- this won't actually help at all as we the plugin system in the critical path imports setuptools / pkg_resources
- I'd rather keep imports at the top

Reply:
Are there other alternatives for parsing versions that don't require `pkg_resources`? 

I remember `distutils` has functionality for that, but @RonnyPfannschmidt was strictly against using it. Wouldn't make sense to use `disutils` just for this bit of functionality? I understand it should be avoided in general, but we are talking about a small utility function of something which is already in the stdlib, not about plugging into `distutils` in a big way or something.

I believe another point was that we require `pkg_resources` anyway because of the plugin system, but given https://github.com/pytest-dev/pluggy/pull/190 it seems doable for us to move away from `pkg_resources` entirely.

Reply:
pkg_resources itself uses `packaging` (vendored)

Reply:
Interesting... perhaps we can consider adding it as dependency if that would mean we can drop `pkg_resources` from `pytest` and `pluggy`?